### DISCUSSION TITLE
[Access Rights] Show certain pages of GH Pages website to certain users

### DISCUSSION THREAD

**Discussion by asalan316** [community/community#75694] [Enterprise]
### Select Topic Area

Question

### Body

Hi there, 

We have a GitHub pages site on our enterprise Github account. We want to show/hide certain pages to particular users. 
Is there any user group restriction concept to achieve the above goal?

Thanks
Arsalan

**Comment by queenofcorgis** [comment]
Hi there! 

Are these pages all under one repository?
